source/_integrations/denonavr.markdown

```diff
@@ -98,6 +98,11 @@ zone2:
   required: false
   default: false
   type: boolean
+update_audyssey:
+  description: Specifies if Audyssey settings should be updated. This can take up to 10 Seconds for some receivers.
+  required: false
+  default: false
+  type: boolean
```

Comment:
This is part of the options flow now?
Additionally, an entity service was introduced to handle this option.

Reply:
Yes it is part of options flow to allow users to update Audyssey settings as part of the update cycle.
Additionally an entity service was added to allow the user to update the settings by them self. I have added the service to the docs.